test/graph-force.html

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
Please use a more specific name to describe what test scene this is created for. Like force-edge-ignoreLayout.html